backend/app/services/public_resource_validation.py

```
@@ -0,0 +1,61 @@
+        return
+
+    invalid_names: list[str] = []
+    for ref in refs:
+        kb_id = ref.get("id") if isinstance(ref, dict) else None
+        kb_name = ref.get("name") if isinstance(ref, dict) else str(ref)
+        kb = (
+            db.query(Kind)
+            .filter(
+                Kind.id == kb_id,
+                Kind.kind == "KnowledgeBase",
+                Kind.is_active == True,
```

Ruff (0.15.18)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)